modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/components/Violations.scss

```diff
@@ -19,7 +19,7 @@ $a11y-panel-width: 320px;
 
 	.applications-menu-modal {
 		.modal-body {
-			padding-right: $a11y-panel-width;
+			padding-right: $a11y-panel-width !important;
```

Comment:
:O

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia4.giphy.com%2Fmedia%2FDBa308wq8XTMs%2Fgiphy.gif&token=chills" data-gitmeme-token="chills"><img src="https://media4.giphy.com/media/DBa308wq8XTMs/giphy.gif" title="Created by gitme.me with /chills"/></a>

Reply:
👀 Well, unfortunately, the DXP doesn't have Sidebar keeping defaults or a framework to handle it better without needing `position: absolute;` since we have a lot of panels using `fixed` too, so we have to adjust this manually unfortunately 😕. Maybe I'm not even covering all the use cases.